src/daft-local-execution/src/pipeline.rs

```diff
@@ -78,7 +78,7 @@ pub fn viz_pipeline(root: &dyn PipelineNode) -> String {
 
 pub fn physical_plan_to_pipeline(
     physical_plan: &LocalPhysicalPlan,
-    psets: &(impl PartitionSet + ?Sized),
+    psets: &(impl PartitionSet<MicroPartition> + ?Sized),
```

Comment:
any reason this isn't generic?

Reply:
it's always over a micropartition set. so we don't want it to be generic here.